Purchases/Public/RCPurchaserInfo.m

```diff
@@ -16,8 +16,6 @@ @interface RCPurchaserInfo ()
 
 @property (nonatomic) NSDictionary<NSString *, NSDate *> *expirationDatesByProduct;
 @property (nonatomic) NSDictionary<NSString *, NSDate *> *purchaseDatesByProduct;
-@property (nonatomic) NSDictionary<NSString *, NSObject *> *expirationDateByEntitlement;
-@property (nonatomic) NSDictionary<NSString *, NSObject *> *purchaseDateByEntitlement;
```

Comment:
@vegaro I removed these because I think, since the schema change essentially means that old caches will get trashed, we can rely on the new data.